src/scripts/httputils/httputils.go

```diff
@@ -7,25 +7,25 @@ import (
 	"net/http"
 )
 
-func SendHTTPRequest(method, url, body string, headers map[string]string) (string, error) {
+func SendHTTPRequest(method, url, body string, headers map[string]string) (string, int, error) {
```

Comment:
I think we can turf this entire file now 🎉

Reply:
make sense